### DISCUSSION TITLE
新版全局代理和绕过大陆怎么设置

### DISCUSSION THREAD

**Discussion by iicemeta** [2dust/v2rayNG#4055] [General]
我想问一下大家，新版v2raynG如何设置全局或绕过大陆？

**Comment by 2dust** [comment]
右上角菜单，导入预设，确定后选择

**Reply by iicemeta** [reply]
成功了，感谢

**Reply by Dade9696** [reply]
为啥还是所在地的定位呀  麻烦问一下怎么改呀

**Reply by iicemeta** [reply]
请提供一下设置截图

**Comment by Dade9696** [comment]
麻烦您看一下 谢谢您

**Reply by iicemeta** [reply]
你的图貌似没传上来

**Reply by iicemeta** [reply]
![Screenshot_2025-02-05-17-56-42-222_com.v2ray.ang-edit.jpg](https://github.com/user-attachments/assets/f66903d5-407a-4388-bbab-21b826e06dbb)

![Screenshot_2025-02-05-17-55-20-202_com.v2ray.ang-edit.jpg](https://github.com/user-attachments/assets/784e4a0c-cfd0-4364-8025-bc9d64786772)

![Screenshot_2025-02-05-17-55-31-075_com.v2ray.ang-edit.jpg](https://github.com/user-attachments/assets/85cf2b60-95c6-40cc-aad9-0ea8d51a8698)

![Screenshot_2025-02-05-17-55-42-668_com.v2ray.ang-edit.jpg](https://github.com/user-attachments/assets/2097f187-4c2a-4f0b-9df8-aa5f692aef6d)

**Comment by Dade9696** [comment]
![微信图片_20250205182212](https://github.com/user-attachments/assets/08f78e1a-0dfe-4006-b163-f061022368a7)
![微信图片_20250205182219](https://github.com/user-attachments/assets/cadefd87-a534-4824-ad4e-8ede1b03fd0e)
![微信图片_20250205182224](https://github.com/user-attachments/assets/83ad9bfe-0b76-4b93-bb3e-f2ef7897c450)
还是这样呀 哥

**Reply by iicemeta** [reply]
你截图浏览器，进ip138.com

**Reply by Dade9696** [reply]
![Uploading 微信图片_20250205184737.jpg…]()

**Reply by iicemeta** [reply]
你图又没传上来

**Reply by Dade9696** [reply]
![微信图片_20250205184737](https://github.com/user-attachments/assets/8004f7cc-f683-4296-9102-bee576939c99)

**Reply by iicemeta** [reply]
全局没问题，你给浏览器定位权限了，把定位关了

**Reply by Dade9696** [reply]
好的谢谢
